schemas/derivatives/SCHEMA.md

```diff
@@ -0,0 +1,69 @@
+| token_address       | The contract address of the token.                 | string |
+| token_symbol        | The symbol of the token.                           | string |
+
+### LP Snapshot
+
+**Description:** Liquidity providers snapshot (one entry for each token in a pool).
+
+| Property          | Description                                           | Type   |
+|-------------------|-------------------------------------------------------|--------|
+| timestamp         | The timestamp of the snapshot.                        | number |
+| chain_id          | Standard chain id.                                    | number |
+| pool_address      | The address of the pool.                              | string |
+| lp_address        | The address of the liquidity provider.                | string |
+| token_index       | The index of the token in the smart contract.         | number |
+| token_address     | The address of the token provided as liquidity.       | string |
+| token_symbol      | The symbol of the token.                              | string |
+| token_amount      | The amount of tokens, decimal normalized.             | number |
+| token_amount_usd  | (Optional) The amount of the provided token in USD.   | number |
+| liquid_token_amount | The amount of tokens supplied that are liquid and usable by users of the protocol, decimal normalized. | number |
+| liquid_token_amount_usd | (Optional) The amount of liquid tokens supplied in USD. | number |
+| total_fees_usd    | (Optional) All fees generated by the LP in the given snapshot in USD. | number |
+| user_fees_usd     | (Optional) The portion of fees accrued to users of the protocol, in USD. | number |
+| protocol_fees_usd | (Optional) The portion of fees accrued to the protocol, in USD. | number | 
+
+### Pool Snapshot
+
+| Property                | Description                                                           | Type   |
+|-------------------------|-----------------------------------------------------------------------|--------|
+| timestamp               | The timestamp of the snapshot.                                        | number |
+| chain_id                | Standard chain id.                                                    | number |
+| pool_address            | The smart contract address of the pool.                               | string |
+| funding_rate            | The funding rate in this pool at the time of the snapshot, as a percentage. | number |
+| fee_rate                | The pool's fee rate, as a percentage.                                 | number |
+| total_value_locked_usd  | (Optional) The total value locked in USD.                             | number |
+| open_interest_longs_usd | (Optional) The open interest of longs in USD.                         | number |
+| open_interest_shorts_usd| (Optional) The open interest of shorts in USD.                        | number |
```

Comment:
i think we need volume data here? 

open_interest_long_usd,open_interest_short_usd  - these are optional, but we should have the underlying token amount if we need to calculate the OI 

also things to note , for OI should it be the OI based on the snapshot?

Reply:
> should it be the OI based on the snapshot?

I imagined it was the current OL / OS at the time of the snapshot.

> i think we need volume data here?

How would you setup the fields for this and what would their description be? `open_long_amount`? or `volume_amount_longs`?

Reply:
volume data is different from OI. OI is the number of positions held by all traders while volumes are the open and shorts closed in a given period. So, we'll need both. Hence:
volumes_usd
open_long_usd
open_short_usd
Pairs on Perps are always TOKEN/USD, so it's easy to get everything adjusted to USD.

Reply:
I added `volume_usd` with description: "The volume of positions opened and closed in the given snapshot in USD." please correct this if it doesn't make sense.